contracts/modules/Payments.sol

```diff
@@ -60,17 +60,16 @@ abstract contract Payments is RouterImmutables {
         if (bips == 0 || bips > 10_000) revert InvalidBips();
         if (token == Constants.ETH) {
             uint256 balance = address(this).balance;
-            uint256 amount = (balance * bips) / FEE_BIPS_BASE;
-            uint256 fixedAmount = (amount * PAYMENT_AMOUNT_BIPS) / FEE_BIPS_BASE;
```

Comment:
Why are you removing `fixedAmount`? What did this used to do

Reply:
it used to send a % of the fee taken to another address, but thats being removed